packages/nodes-base/nodes/Discord/Discord.node.ts

```diff
@@ -23,85 +24,247 @@ export class Discord implements INodeType {
 		},
 		inputs: ['main'],
 		outputs: ['main'],
-		properties: [
+		properties:[
 			{
 				displayName: 'Webhook URL',
 				name: 'webhookUri',
 				type: 'string',
 				typeOptions: {
 					alwaysOpenEditWindow: true,
 				},
+				required: true,
 				default: '',
-				description: 'The webhook url.',
+				placeholder: 'https://discord.com/api/webhooks/ID/TOKEN',
 			},
 			{
-				displayName: 'Text',
-				name: 'text',
+				displayName: 'Content',
+				name: 'content',
```

Comment:
Changing this name actually causes this to become a breaking change; we should keep the `name` the same. The label (`displayName`) could be changed, but it's better to keep it as it was.